File: test/corpus/languages/cpp/fixture/test_main.cpp
#include "user.hpp"

void test_create_sample_user() {
    auto user = example::createSampleUser();
}

void test_user_repository() {
    auto storage = std::make_unique<example::MemoryStorage>();
    example::UserRepository repo(std::move(storage));
    auto user = example::createSampleUser();
    repo.addUser(user);
}

File: test/corpus/languages/python/fixture/test_main.py
from main import create_sample_user, UserRepository


def test_create_sample_user():
    user = create_sample_user()
    assert user.name == "John"


def test_user_repository():
    repo = UserRepository()
    user = create_sample_user()
    repo.add_user(user)
    assert repo.count_users() == 1
